### ISSUE TITLE
Prompt text field memory

### ISSUE THREAD

**Issue by sebrojas14** [MohammadYounes/AlertifyJS#59]
The prompt dialog keeps the value entered for the user for ever, the value param in the promt method is ignored after the user has filled the text field. Apparently, only the first time I call alertify.prompt an instance is creted, so the build methos is called. If I call alertify.prompt, only the main method is called.

**Comment by MohammadYounes** [comment]
> The prompt dialog keeps the value entered for the user for ever, the value param in the promt method is ignored after the user has filled the text field.

I traced the issue and the bug is caused by [this line](https://github.com/MohammadYounes/AlertifyJS/blob/master/src/js/prompt.js#L154) which should be:

 ```js
  this.settings.value = input.value;
```
Instead of:
```js
this.value = input.value;
```

> Apparently, only the first time I call `alertify.prompt` an instance is created, so the build method is called. If I call `alertify.prompt`, only the main method is called.

Yes, its made to be a singleton and the `build` method is called only once upon dialog creation.

Thanks.

**Comment by sebrojas14** [comment]
No, that's not the solution, after that fix the onok function (function (evt, value)) returns undefined for the value parameter

**Comment by MohammadYounes** [comment]
2 [lines after](https://github.com/MohammadYounes/AlertifyJS/blob/master/src/js/prompt.js#L156) after, `this.value` is passed again, replace it with `this.settings.value`.

**Comment by MohammadYounes** [comment]
Sorry for the delay, fixed in `1.4.1`

Thanks!

**Comment by sebrojas14** [comment]
Thank you, seems working
